.travis.yml

```diff
@@ -3,6 +3,7 @@ python:
 - '2.7'
 - '3.4'
 - '3.5'
+- '3.6'
```

Comment:
should this have 3.7 too?

Reply:
this list causes Travis to run 1 job per version, right?
how does this interact with tox which normally tests many versions?
OK, I see https://github.com/tox-dev/tox-travis explains tox looks at TRAVIS_PYTHON_VERSION and runs only appropriate version, nice.

Reply:
The support of 3.7 on Travis is still problematic, I mentioned it in the PR description.
When added it fails like https://travis-ci.org/ManageIQ/manageiq-api-client-python/jobs/469887540
There is workaround available, but I'd prefer to wait until it's supported the same way the other versions are.